### DISCUSSION TITLE
Show entities in areas

### DISCUSSION THREAD

**Discussion by frenck** [home-assistant/frontend#7915] [Configuration]
With Home Assistant 1.0.0b0 and newer, it became possible to assign entities to an area.

This is a very helpful and cool feature; as it allows for targeting areas better and make them more useful. However, the area overview doesn't show this:

![image](https://user-images.githubusercontent.com/195327/101265845-2d2d8e80-374a-11eb-8917-fbdeba165e15.png)

An entity of a device or an entity without a device; isn't showing up in the area panel.

I think it should be considered to add this.

Some considerations:

- An Device can be assigned to an area, but one of the entities of a device can be assigned to a different area. Does this mean that this device will now show up in both areas?

- Automations are displayed based on their entity usage; However, automations themselves can be assigned to an area as well, those don't show up in the automations list.

**Comment by Masterz69** [comment]
Agree - some integrations does NOT create Devices, but only Entities.
As well as Entity can be individually assigned to other than Device area.
